src/pk/rsa/rsa_make_key.c

```diff
@@ -32,10 +32,6 @@ int rsa_make_key(prng_state *prng, int wprng, int size, long e, rsa_key *key)
    LTC_ARGCHK(ltc_mp.name != NULL);
    LTC_ARGCHK(key         != NULL);
 
-   if ((size < (MIN_RSA_SIZE/8)) || (size > (MAX_RSA_SIZE/8))) {
```

Comment:
considering the fact that `size` is `int` we should at least check `LTC_ARGCHK(size > 0)`